.gitignore

```diff
@@ -7,3 +7,10 @@ tests/venv
 # IDE metadata
 .idea
 .project
+
+.DS_Store
```

Comment:
nit: I think you can just ignore `**.DS_Store`

context: https://www.quora.com/Does-gitignore-file-only-work-on-current-folder